app/main/views/g_cloud.py

```diff
@@ -34,6 +34,7 @@
 
 
 PROJECT_CREATED_MESSAGE = Markup("""Your new project has been created.""")
+PROJECT_ENDED_MESSAGE = Markup("""Your search has been ended. You can now download your shortlist.""")
```

Comment:
Is this real content? Seems like a weird sentence.

Reply:
Placeholder, real thing can edit for Karl, agree first sentence should probably have been "marked as complete" or something, but somehow be consistent with "ending".. shrug :)